README.md

```diff
@@ -1,6 +1,6 @@
-# Awesome Book
+# React.js Calculator
```

Comment:
- [ ] As it is a requirement for this project, please kindly update the your readme file following this [template](https://github.com/microverseinc/readme-template). This will enable visitors including potential recruiters to quickly have information about the project.

Reply:
Sure, Thanks!!